pkgs/by-name/op/opencode/package.nix

```diff
@@ -106,17 +106,15 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     outputHashMode = "recursive";
   };
 
-  models-dev-data = fetchurl {
-    url = "https://models.dev/api.json";
-    sha256 = "sha256-igxQOC+Hz2FnXIW/S4Px9WhRuBhcIQIHO+7U8jHU1TQ=";
-  };
-
-  nativeBuildInputs = [ bun ];
+  nativeBuildInputs = [
+    bun
+    models-dev
```

Comment:
Is this really required during the build of this derivation? Or only at runtime ?

Reply:
Yes, `packages/opencode/src/provider/models-macro.ts` is executed only during the build process, which is why we need to patch it.

Reply:
No, I don’t believe so. I couldn’t find any documentation specifically about `env`. Runtime environment variables are usually added using `makeWrapper` (see https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper). From what I’ve seen, `env.<variable>` is primarily used for `buildPhase` environment variables (e.g., https://nixos.org/manual/nixpkgs/stable/#var-go-CGO_ENABLED).